chromium_src/chrome/elevation_service/elevator.cc

```diff
@@ -12,10 +12,11 @@
 
 #include "base/win/windows_types.h"
 #include "brave/components/brave_vpn/common/buildflags/buildflags.h"
+#include "chrome/common/channel_info.h"
```

Comment:
does `//chrome/elevation_service` depend on `//chrome:common`?

Reply:
Hmm, it's weird that it's not in deps list but didn't get complain about this.
BTW, I found that `install_static::GetChromeChannel()` could be used here instead to get channel
and it's already in deps list(`//chrome/install_static:install_static_util`). Updated - https://github.com/brave/brave-core/pull/21549/commits/6fb008182b7800b8be52669f1478b2c78636bb93

Reply:
Filed issue https://github.com/brave/brave-browser/issues/35772 for this.